documentation/api/query/v4/facts.markdown

```diff
@@ -80,7 +80,7 @@ Subquery against `/fact-contents` to get all remotely-authenticated trusted fact
 
     curl -X GET http://localhost:8080/v4/facts --data-urlencode 'query=["in", ["name","certname"],
       ["extract",["name","certname"],
-        ["select-fact-contents", ["~>", "path", [".*", "authenticated"]]]]]'
+        ["select_fact_contents", ["~>", "path", [".*", "authenticated"]]]]]'
```

Comment:
spoke to @jdwelch about this, consensus is select_fact_contents and fact-contents